### ISSUE TITLE
Notification Panel

### ISSUE THREAD

**Issue by Linkaloo** [UNLV-CS472-672/2024-S-GROUP1-Roadwatch#96]
## Summary
Create the notification panel and link with the notification icon.
No need for notification updates right now.

## Technical Details
Client
Make a component for each notification to use.
Do not route to a notifications page, instead see if you can slide out some notification container.

## Acceptance Criteria
- Notification component
- - Title of notification
- - Any comment pertaining to it
- - Time since posted
- When notification bell is pressed in home page it brings up the notification
- X icon to close the container

## Figma
![image](https://github.com/UNLV-CS472-672/2024-S-GROUP1-Roadwatch/assets/24483977/fac9fa78-bd51-4026-9aec-baa98ec1ebc3)
![image](https://github.com/UNLV-CS472-672/2024-S-GROUP1-Roadwatch/assets/24483977/2fdcbc6e-27b0-4b86-aaf8-09a4eab70b5d)

**Comment by Linkaloo** [comment]
So we have that notification panel, but we can also explore little popup notifications like these: https://www.npmjs.com/package/react-toastify

**Comment by KingJordan152** [comment]
> So we have that notification panel, but we can also explore little popup notifications like these: https://www.npmjs.com/package/react-toastify

I don't think we need this considering we have working notifications now. Might be easier and less cluttered to stick with that for the time being.

**Comment by KingJordan152** [comment]
Explore populating this page with `getNotifications()` from the `ServiceWorkerRegistration` interface: https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerRegistration/getNotifications.

If that doesn't work, we might have to store sent notifications in the `User` schema.
